<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MDF Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Convert">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>380</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="Belgium"/>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Click to browse file to convert">Click and select MDF file for conversion</string>
    </property>
    <property name="locale">
     <locale language="English" country="Belgium"/>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="Belgium"/>
    </property>
    <property name="text">
     <string>File Channel List</string>
    </property>
    <property name="buddy">
     <cstring>channelList</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="Belgium"/>
    </property>
    <property name="text">
     <string>File List</string>
    </property>
    <property name="buddy">
     <cstring>FileList</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="Belgium"/>
    </property>
    <property name="text">
     <string>Selected Channel List</string>
    </property>
    <property name="buddy">
     <cstring>SelectedChannelList</cstring>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>116</width>
      <height>291</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="ConvertSelect">
       <item>
        <widget class="QRadioButton" name="matlab">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Matlab .mat</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="netcdf">
         <property name="text">
          <string>netCDF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="hdf5">
         <property name="text">
          <string>HDF5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="csv">
         <property name="text">
          <string>CSV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="excel">
         <property name="text">
          <string>Excel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <layout class="QVBoxLayout" name="Resample">
       <item>
        <widget class="QCheckBox" name="resample">
         <property name="locale">
          <locale language="English" country="Belgium"/>
         </property>
         <property name="text">
          <string>Resample</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="resampleValue">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="LabFile_2">
       <item>
        <widget class="QPushButton" name="LabFileBrowse">
         <property name="locale">
          <locale language="English" country="Belgium"/>
         </property>
         <property name="text">
          <string>Lab file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LabFile">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QCheckBox" name="MergeFiles">
     <property name="locale">
      <locale language="English" country="Belgium"/>
     </property>
     <property name="text">
      <string>MergeFiles</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>681</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QListWidget" name="FileList">
     <property name="locale">
      <locale language="English" country="Belgium"/>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="channelList">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="locale">
      <locale language="English" country="Belgium"/>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="SelectedChannelList">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="locale">
      <locale language="English" country="Belgium"/>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionBrowse">
   <property name="text">
    <string>Browse</string>
   </property>
   <property name="toolTip">
    <string>Open MDF file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>resample</sender>
   <signal>clicked()</signal>
   <receiver>resampleValue</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>747</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>789</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
